diff --git a/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv4.js b/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv4.js
index edab5a8..2359eb8 100644
--- a/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv4.js
+++ b/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv4.js
@@ -2,8 +2,10 @@
 import * as common from './common.js';
 import * as constants from './v4/constants.js';
 import { AddressError } from './address-error.js';
-import { BigInteger } from 'jsbn';
+import jsbn from 'jsbn'
 import { sprintf } from 'sprintf-js';
+
+const { BigInteger } = jsbn;
 /**
  * Represents an IPv4 address
  * @class Address4
diff --git a/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv6.js b/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv6.js
index ae9f334..eebc6d2 100644
--- a/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv6.js
+++ b/node_modules/@achingbrain/ip-address/dist/esm/lib/ipv6.js
@@ -7,8 +7,10 @@ import * as helpers from './v6/helpers.js';
 import { Address4 } from './ipv4.js';
 import { ADDRESS_BOUNDARY, possibleElisions, simpleRegularExpression, } from './v6/regular-expressions.js';
 import { AddressError } from './address-error.js';
-import { BigInteger } from 'jsbn';
+import jsbn from 'jsbn';
 import { sprintf } from 'sprintf-js';
+const { BigInteger } = jsbn;
+
 function assert(condition) {
     if (!condition) {
         throw new Error('Assertion failed.');
